bayesian/api_v1.py

```diff
@@ -1076,6 +1076,36 @@ def get(runtime):
         return response, 200
 
 
+class CoreDependencies(ResourceWithSchema):
+    """Implementation of Blank booster /get-core-dependencies REST API call."""
+
+    method_decorators = [login_required]
+
+    @staticmethod
+    def get(runtime):
+        """Handle the GET REST API call."""
+        try:
+            resolved = list()
```

Comment:
`resolved = []` is more idiomatic and faster as well https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better